app/controllers/api/v1/base_api_controller.rb

```diff
@@ -0,0 +1,8 @@
+module Api
+  module V1
+    class BaseApiController < ApplicationController
```

Comment:
I would combine the ApiController and this one, why would you leave both separated?